src/main/java/core/basesyntax/service/impl/ReaderImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.Reader;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReaderImpl implements Reader {
+
+    @Override
+    public List<String[]> read(String filePath) {
+        List<String[]> linesArray = new ArrayList<>();
+        String line;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(filePath))) {
+            bufferedReader.readLine();
+            while ((line = bufferedReader.readLine()) != null) {
+                linesArray.add(line.split(","));
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file: " + e);
```

Comment:
In the catch block, when rethrowing the exception, include the original exception object so that the stack trace is preserved. Change the catch block to throw new RuntimeException("Can't read file: " + filePath, e);